airflow/serialization/serialized_objects.py

```diff
@@ -878,6 +920,34 @@ def from_dict(cls, serialized_obj: dict) -> 'SerializedDAG':
             raise ValueError(f"Unsure how to deserialize version {ver!r}")
         return cls.deserialize_dag(serialized_obj['dag'])
 
+    @classmethod
+    def _serialize_dag_params(cls, dag_params: ParamsDict):
+        """Serialize Params dict for a DAG"""
+        serialized_params = {}
+        for k, v in dag_params.items():
+            # To support backward compatability with some operators, we would let str params
+            # serialized as well, after converting them into a shallow Param object.
+            if v.__class__.__name__ == 'str':
```

Comment:
```suggestion
            if isintance(v, 'str'):
```

Reply:
```suggestion
            if isinstance(v, str):
```

I think you mean Kaxi

Reply:
oof yes